lib/MultiStats.js

```diff
@@ -2,70 +2,63 @@
 	MIT License http://www.opensource.org/licenses/mit-license.php
 	Author Tobias Koppers @sokra
 */
-var Stats = require("./Stats");
+"use strict";
 
-function MultiStats(stats) {
-	this.stats = stats;
-	this.hash = stats.map(function(stat) {
-		return stat.hash;
-	}).join("");
-}
+const Stats = require("./Stats");
 
-MultiStats.prototype.hasErrors = function() {
-	return this.stats.map(function(stat) {
-		return stat.hasErrors();
-	}).reduce(function(a, b) {
-		return a || b;
-	}, false);
-};
+class MultiStats extends Stats {
```

Comment:
Tried to avoid duplication of `toString` & `optionOrFallback` from Stats - which results with failing tests in MultiCompiler. i can resolve it with the duplication of those two - unless someone has a better idea?

We can export them both from Stats and require them over here?